docs/attributes-registry/gen-ai.md

```diff
@@ -13,8 +13,8 @@ This document defines the attributes used to describe telemetry in the context o
 | Attribute                          | Type     | Description                                                                                      | Examples                                                                | Stability                                                        |
 | ---------------------------------- | -------- | ------------------------------------------------------------------------------------------------ | ----------------------------------------------------------------------- | ---------------------------------------------------------------- |
 | `gen_ai.completion`                | string   | The full response received from the GenAI model. [1]                                             | `[{'role': 'assistant', 'content': 'The capital of France is Paris.'}]` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
-| `gen_ai.operation.name`            | string   | The name of the operation being performed.                                                       | `chat`; `completion`                                                    | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
-| `gen_ai.prompt`                    | string   | The full prompt sent to the GenAI model. [2]                                                     | `[{'role': 'user', 'content': 'What is the capital of France?'}]`       | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `gen_ai.operation.name`            | string   | The name of the operation being performed. [2]                                                   | `chat`; `completions`                                                   | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
why `completions` and not `completion`?
https://docs.anthropic.com/en/api/complete

Reply:
Just to stay  consistent with OpenAI name [`completions` (plural)](https://platform.openai.com/docs/api-reference/completions)
Cohere calls it `generate` - https://docs.cohere.com/reference/generate-1

If it's `completion` then it does not align with anyone, which is also fine (since it's legacy everywhere), but then I'd then prefer not to define it at all

Reply:
We agreed on `text_completion` in our weekly meeting